src/spid-sdk.js

```diff
@@ -11,6 +11,28 @@ var
     cache = require('./spid-cache'),
     talk = require('./spid-talk');
 
+var sessionCallbacks = (function sessionRequests() {
```

Comment:
No, if so I would have put it in its own module ;-)
But seriously, maybe. But I don´t think it makes sense out of this context.